ghost/core/core/server/services/milestone-emails/service.js

```diff
@@ -26,14 +26,41 @@ module.exports = {
             } = require('@tryghost/milestone-emails');
             const config = require('../../../shared/config');
             const milestonesConfig = config.get('milestones');
-            const {GhostMailer} = require('../mail');
+            const hostSettings = config.get('hostSettings');
+            const events = require('../../lib/common/events');
 
-            const mailer = new GhostMailer();
             const repository = new InMemoryMilestoneRepository();
             const queries = new MilestoneQueries({db});
 
+            const slackEnabled = hostSettings?.milestones?.enabled && hostSettings?.milestones?.url;
+
+            if (slackEnabled) {
+                const {SlackNotifications} = require('@tryghost/slack-notifications');
+                const slackNotifications = new SlackNotifications({
```

Comment:
I think this should be instantiated and created in it's own service in `/ghost/core/core/server/services/slack-notifications/service.js`